services/api/kalvi/api/urls/__init__.py

```diff
@@ -12,4 +14,24 @@
         SignInEndPoint.as_view(),
         name="kalvi-sign-in",
     ),
+    path(
+        "profile/",
+        UserProfileView.as_view(),
+        name="UserDetail",
+    ),
+    path(
+        "update-password/",
+        UserChangePasswordView.as_view(),
+        name="password-change",
+    ),
+    path(
+        "send-reset-password-email/",
+        SendPasswordResetEmailView.as_view(),
+        name="SendResetEmail",
+    ),
+    path(
+        "reset-password/<uid>/<token>/",
+        UserPasswordResetView.as_view(),
+        name="ResetPasswordThroughMail",
```

Comment:
Lets be consistent with the naming convention.

Reply:
decided to go with "-" not camelCase .